xml/cap_depl_azure.xml

```diff
@@ -235,11 +235,17 @@ cap-aks</screen>
    List the &kube; versions currently supported by AKS (see
    <link xlink:href="https://docs.microsoft.com/en-us/azure/aks/supported-kubernetes-versions"/>
    for more information on the AKS version support policy). When creating your
-   cluster in the next step, specify a &kube; version for consistent
-   deployments:
+   cluster in the next step, specify the newest supported &kube; version:
   </para>
 
-<screen>&prompt.user;az aks get-versions --location $REGION --output table</screen>
+<screen>&prompt.user;az aks get-versions --location $REGION --output table
+ KubernetesVersion    Upgrades
```

Comment:
Are we planning to keep this table updated or are we ok with leaving updating periodically? I ask because the available version changes quite often.